src/examples/table/TableSelectRows.tsx

```diff
@@ -120,7 +120,7 @@ const Example = () => {
                       setFocusedRowIndex(index);
                     }}
                   >
-                    <Label hidden>Select ticket</Label>
+                    <Label hidden>Select ticket #{index}</Label>
```

Comment:
I think we should be even more descriptive still since "#1" would still not provide enough info to an AT user about the row being referenced. How about:

```suggestion
                    <Label hidden>Select ticket for Fruit #{index}</Label>
```

Reply:
Sounds good. Changes have been made.